src/Home/index.js

```diff
@@ -59,23 +81,85 @@ export const Home = () => (
 		<Header />
 		<Main>
 			<Content>
-				<Banner>
-					<BannerTitle isWide={false}>
-						Revolutionizing music with the blockchain
-					</BannerTitle>
-					Listen to Artist of the week<span>
-						<br />
-					</span>
-					<BannerText>
-						100% free streaming for listeners, industry best compensation for
-						musicians.
-					</BannerText>
-					<PrimaryButton to="https://musicoin.org/welcome" target="_blank">
-						Join the revolution
-					</PrimaryButton>
-					<ArrowButton to="/how-it-works">How it works</ArrowButton>
-				</Banner>
+				<Media marginTop={'125px'}>
+					<MediaImage src={mobileAppImage} order={0} width={477} height={450} />
+					<MediaContent>
+						<TertiaryHeading level={2} isPretty={true} color={'#FFFFFF'}>
+							Music For All
+						</TertiaryHeading>
+						<p>
+							95% of internet users can't listen to music online easily today
```

Comment:
@JoMo1970 please follow Pam's most recent text for this part. Then we can merge.